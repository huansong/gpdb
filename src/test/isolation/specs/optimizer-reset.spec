# Reset optimizer setting for the isolation database

session s1
setup { alter database isolation_regression reset optimizer; }
step dummy { select 1; }
