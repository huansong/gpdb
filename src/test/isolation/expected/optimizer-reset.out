Parsed test spec with 1 sessions

starting permutation: dummy
step dummy: select 1;
?column?
--------
       1
(1 row)

